prometheus_client/metrics_core.py

```diff
@@ -58,6 +58,15 @@ def __repr__(self):
             self.samples,
         )
 
+    def restricted_metric(self, names):
```

Comment:
Do you think there is a good use case for using this function externally? If not perhaps it should be `_restricted_metric` so we could easily change/remove it later.

Reply:
Fixed. Good catch!